charts/fleet/templates/deployment.yaml

```diff
@@ -17,6 +17,14 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        {{- if .Values.proxy }}
+        - name: HTTP_PROXY
+          value: {{ .Values.proxy }}
+        - name: HTTPS_PROXY
+          value: {{ .Values.proxy }}
+        - name: NO_PROXY
+          value: {{ .Values.noProxy }}
+        {{- end }}
```

Comment:
Some are proxy, others are noProxy. Is this fine?

Reply:
Followed the rancher pattern in here.

Reply:
https://github.com/rancher/rancher/blob/8455c2919aec5acfc03fdf4a747bc5ad5d983899/chart/templates/deployment.yaml#L88-L95